gnomad_qc/v4/resources/basics.py

```diff
@@ -157,14 +182,21 @@ def _ukbb_root_path() -> str:
 
 
 # List of samples to exclude from QC due to withdrawn consents
-# This is the file with the most up to date sample withdrawals we were sent. File downloaded on 8/16/21
-ukbb_excluded_samples_path = f"{_ukbb_root_path()}/w26041_20210809.csv"
+ukbb_excluded_26041_path = f"{_ukbb_root_path()}/w26041_20210809.csv"
```

Comment:
Thinking about this more I think we want to make a VersionedTableResource so we could turn these into HTs already and then use something like:

```
ukbb_excluded = VersionedMatrixTableResource(
    default_version="31063_20220222",
    versions={
        "26041_20210809": TableResource(
            path=f"{_ukbb_root_path()}/w26041_20210809.ht"
        ),
        "31063_20220222":f"{_ukbb_root_path()}/ukb31063.withdrawn_participants.20220222.ht"
    },
)
```
There is also the ability to use the `import_func` and `import_args` if we want to, but I'm not sure that is needed honestly, because you would need to make something like this available in the script: https://github.com/broadinstitute/gnomad_methods/blob/541080e0f62296af63014b26dbb311bb38a65cec/gnomad/resources/import_resources.py#L89. But if we turn them into HTs once, we don't need to keep loading them from the csv. You can add a note for how the HT was created